CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.2.8] - 2017-11-01
+- Massive topic deletion while switching timelines is now made with small interval between deletions
```

Comment:
There should be additional header like `### Fixed` or `### Changed`

Reply:
done